src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -510,7 +510,7 @@ def generate_flow_meta_dict_by_file(entry: str, source: str = None, path: str =
         flow_meta["inputs"] = {}
         for k, v in tool.inputs.items():
             # We didn't support specifying multiple types for inputs, so we only take the first one.
-            flow_meta["inputs"][k] = {"type": v.type[0].value}
+            flow_meta["inputs"][k] = {"type": v.type[0].value, "default": v.default}
```

Comment:
Will we have a default None for inputs without default value?

Reply:
Refine, don't include default if it is not specified.